src/sinol_make/commands/run/__init__.py

```diff
@@ -203,9 +203,59 @@ def compile(self, solution):
             return False
 
 
-    def execute_oiejq(self, command, result_file, output_file, answer_file, time_limit, memory_limit):
+    def check_output_diff(self, output_file, answer_file):
+        """
+        Checks if the output file and the answer file are the same.
+        Returns True if they are the same, False otherwise.
+        """
+        return not os.system("diff -q -Z %s %s >/dev/null" % (output_file, answer_file))
```

Comment:
If we can avoid using bash commands, we should not use them and instead use the Python alternatives. We don't know on what kind of shells the user is going to run sinol-make. Some commands differ between shells or operating systems. Unless we have a guarantee that for each operating system the `diff` command exists, accepts those arguments, has the same definition of those arguments and returns the same, it's not safe to use the `diff` command.

Reply:
I added a function that should work the same as `diff`: b536c3ec6493d57647886b0b0f99ae513c2786c2